### PR TITLE
Community [MARXAN-455] [MARXAN-457]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

- Community static page. 
- Added Column components to statics layout and refactor About page including Column.
- Detail Published Project.
- Share on detail page. 

*Facebook only shares valid links, so it will fail on local*

### Designs

https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=4668%3A0

### Feature relevant tickets

https://vizzuality.atlassian.net/secure/RapidBoard.jspa?rapidView=38&projectKey=MARXAN&modal=detail&selectedIssue=MARXAN-457&quickFilter=71

---

### CODE REVIEW

File: app/utils/k-formatter.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const kFormatter = (num) => {
```
Comment (mbarrenechea) [inline app/utils/k-formatter.js:1]: We don't need a kFormatter. What would happen if you have millions or billions? It's better if we use a number formatter library. As we have already installed `d3` in the app we have access to d3-format

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: app/utils/k-formatter.js (hunk 1 above)
Comment (mbarrenechea) [reply app/utils/k-formatter.js:1]:
```
import { format } from 'd3-format';

format('.3s')(value)
```

File: app/.storybook/preview.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const parameters = {
       order: [
         'Intro',
         'Docs', ['Install', 'Deploy', 'Authentication', 'Fetching', 'Media', 'Tests'],
-        'Components'
+        'Components', ['Avatar', 'Breadcrum', 'Button', 'ConfirmationPrompt', 'Features', 'Forms', 'GapAnalysis', 'Icon', 'InfoButton','Loading', 'Map', 'Modal', 'ProgressBar', 'Projects', 'Scenarios', 'Search', 'Steps', 'Table', 'Tabs', 'Tag', 'Toast', 'Tooltip'],
```
Comment (mbarrenechea) [inline app/.storybook/preview.js:13]: Why are you adding this order if at the end you are sorting them by name? Could you remove it?

File: app/components/projects/published-item/component.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  argTypes: {},
+};
+
+const Template: Story<PublishedItemProps> = ({ ...args }: PublishedItemProps) => (
+  <Item {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  id: 1,
```
Comment (mbarrenechea) [inline app/components/projects/published-item/component.stories.tsx:17]: Can you check the default args you are setting for this item? I think you need to update them

File: app/layout/about/utilities/component.tsx [hunk 4]
```diff
@@ -13,19 +13,12 @@ export const AboutUtilities: React.FC<AboutUtilitiesProps> = () => {
...
-            and Marxan with Connectivity on a stable Azure cloud environment.
-            You can use this platform to:
-          </p>
-          <ButtonLink />
-        </div>
+        <Column
+          title="What can I do with this Marxan planning platform?"
+          subtitle="This platform hosts the Marxan software suite which includes Marxan, Marxan with Zones, and Marxan with Connectivity on a stable Azure cloud environment. You can use this platform to:"
+          caption="Learn more"
+          href="/"
```
Comment (mbarrenechea) [inline app/layout/about/utilities/component.tsx:20]: the `learn more` button is going to send me to the home page?

File: app/layout/community/component.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+          caption="Explore projects"
+          href="community/projects"
+        />
+        <Column
+          title="Join the community and stay updated"
+          subtitle="Receive the latest Marxan updates and connect with other users to mobilize for action."
+          description="Have questions about Marxan or suggestions for how to strengthen the platform? How about an idea for a bold new partnership around data use? We want to hear from you. Join us in the movement to build a more sustainable future."
+          caption="Learn more"
+          href="/"
```
Comment (mbarrenechea) [inline app/layout/community/component.tsx:27]: the learn more button is going to send me to the home page?

File: app/layout/community/component.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+export const CommunityInfo: React.FC<CommunityInfoProps> = () => {
+  return (
+    <Wrapper>
+      <div className="grid w-full max-w-5xl grid-cols-2 py-16 mx-auto border-t border-white gap-x-36 border-opacity-20">
+        <Column
+          title="Explore the projects shared by the community"
+          subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore."
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+          caption="Explore projects"
+          href="community/projects"
```
Comment (mbarrenechea) [inline app/layout/community/component.tsx:20]: use absolute paths. is this an internal link? I see you have a prop for setting this

File: app/layout/community/published-projects/component.tsx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+          aria-label="Search"
+          onChange={onChangeSearchDebounced}
+        />
+        <h3 className="pt-10 text-2xl font-heading">
+          Projects published (
+          {publishedProjectsData.length}
+          )
+        </h3>
+        {publishedProjectsIsFetching && (
+        <div className="flex items-center justify-center py-12">
```
Comment (mbarrenechea) [inline app/layout/community/published-projects/component.tsx:63]: indentation

File: app/layout/statics/button-link/component.tsx [hunk 8]
```diff
@@ -1,21 +1,45 @@
...
+  );
+
+  return (
+    <>
+      {internal && (
+        <a className="flex flex-row items-center cursor-pointer" href={href}>
+          {children}
+        </a>
+      )}
+      {!internal && (
```
Comment (mbarrenechea) [inline app/layout/statics/button-link/component.tsx:35]: when you say internal, you mean external right? Change the name of this prop to `external`

File: app/layout/statics/contact/component.tsx [hunk 9]
```diff
@@ -19,7 +19,7 @@ export const StaticContact: React.FC<StaticContactProps> = () => {
             Help us strengthen Marxan.
           </p>
           <div className="w-44">
-            <Button theme="transparent-black" size="lg" type="submit" className="w-full">
+            <Button href="mailto:marxancloud@gmail.com" theme="transparent-black" size="lg" type="submit" className="w-full">
```
Comment (mbarrenechea) [inline app/layout/statics/contact/component.tsx:22]: It should open in a new tab

File: app/layout/community/published-projects/component.tsx [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+export const CommunityProjectsList: React.FC<CommunityProjectsListProps> = () => {
+  const {
+    data: publishedProjectsData,
+    isFetching: publishedProjectsIsFetching,
+    isFetched: publishedProjectsIsFetched,
+  } = usePublishedProjects();
+
+  const { search } = useSelector((state) => state['/community/projects']);
```
Comment (mbarrenechea) [inline app/layout/community/published-projects/component.tsx:33]: There is an error when I try to get to this page related to this. Did you fix it in this PR or in the other one?

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: app/components/projects/published-item/component.stories.tsx (hunk 3 above)
Comment (anamontiaga) [reply app/components/projects/published-item/component.stories.tsx:17]: Updated and rendered on page.

File: app/layout/community/published-projects/component.tsx (hunk 10 above)
Comment (anamontiaga) [reply app/layout/community/published-projects/component.tsx:28]: It will be on next PR (detail of published project)

**Pull request closed without merging**